doc/modules/ensemble.rst

```diff
@@ -1033,3 +1032,90 @@ The following example shows how to fit the VotingRegressor::
 .. topic:: Examples:
 
   * :ref:`sphx_glr_auto_examples_ensemble_plot_voting_regressor.py`
+
+.. _stacking:
+
+Stacked generalization
```

Comment:
This is a subsection of the `VotingRegressor` part (1.11.6.1). Shouldn't this be in its own section (1.11.7) ?

Also nit: I think `Stacking` is more common and might be easier for the eye to catch